scss/_product-page.scss

```diff
@@ -159,6 +159,7 @@ $decalTop:48px;
 .accordion dd.active>a {
     background: #ffffff;
     border-radius: 8px 8px 0 0 ;
+    border-left: 6px solid #ccc;
```

Comment:
Could you remove the border from this pull request, so that we can first merge the other changes? For the border, it's likely there will be some discussion on how it should look.